Mathlib/Data/Set/NAry.lean

```diff
@@ -430,4 +430,20 @@ lemma image2_right_identity {f : α → β → α} {b : β} (h : ∀ a, f a b =
   rw [image2_singleton_right, funext h, image_id']
 #align set.image2_right_identity Set.image2_right_identity
 
+theorem image2_inter_union_subset {f : α → α → β} {s t : Set α} (hf : ∀ a b, f a b = f b a) :
+    image2 f (s ∩ t) (s ∪ t) ⊆ image2 f s t :=
+  by
```

Comment:
nit: wrap to the previous line like the `start_port.sh` script would, same below